src/utils/__tests__/dockerDirManager.spec.ts

```diff
@@ -280,4 +338,36 @@ describeUnix('LimaBackend', () => {
       expect(consoleMock).not.toHaveBeenCalled();
     });
   });
+
+  describe('credHelperWorking', () => {
+    let fakeProcess: childProcess.ChildProcess;
+    let spawnMock: jest.SpyInstance<childProcess.ChildProcess, [message?: any, ...optionalArgs: any[]]>;
```

Comment:
```suggestion
    let spawnMock: jest.SpiedFunction<typeof childProcess.spawn>;
```
Just to let things be explicitly correct.